### PR TITLE
2-character usernames can now login

### PR DESCRIPTION
**Pull request by ImALuckyGuy**
[16:18:03] [Thread-112/INFO]: UUID of player 5m is 1fe6be27-3283-3118-911c-fbb124d5d1b1
[16:18:03] [Server thread/INFO]: [TotalFreedomMod] Obtaining UUID for new player: 5m
[16:18:03] [Server thread/INFO]: Disconnecting com.mojang.authlib.GameProfile@4ccce6e6[id=1fe6be27-3283-3118-911c-fbb124d5d1b1,name=5m,properties={},legacy=false] (/185.43.110.xx:59584): Your username is an invalid length (must be between 3 and 20 characters long).
[16:18:03] [Server thread/INFO]: com.mojang.authlib.GameProfile@4ccce6e6[id=1fe6be27-3283-3118-911c-fbb124d5d1b1,name=5m,properties={},legacy=false] (/185.43.110.xx:59584) lost connection: Your username is an invalid length (must be between 3 and 20 characters long).

Turns out that name is an actual premium username and they couldn't join.

### CODE REVIEW

**Comment by ImALuckyGuy** [comment]
Darth, you silly, this isn't an issue, this is a PR. :P

**Comment by JeromSar** [comment]
@ImALuckyGuy Whoopsies.

File: src/me/StevenLawson/TotalFreedomMod/TFM_ServerInterface.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static void handlePlayerLogin(PlayerLoginEvent event)
         final UUID uuid = TFM_UuidManager.newPlayer(player, ip);
 
         // Perform username checks
-        if (username.length() < 3 || username.length() > TotalFreedomMod.MAX_USERNAME_LENGTH)
+        if (username.length() < 2 || username.length() > TotalFreedomMod.MAX_USERNAME_LENGTH)
         {
             event.disallow(Result.KICK_OTHER, "Your username is an invalid length (must be between 3 and 20 characters long).");
```
Comment (Scuph) [inline src/me/StevenLawson/TotalFreedomMod/TFM_ServerInterface.java]: it still says (must be between 3 and 20 characters), fix that.

**Comment by JeromSar** [comment]
@StevenLawson @Wild1145 Do we want 2-character usernames to be able to log in?

**Comment by Wild1145** [comment]
@JeromSar I think if Minecraft supports them then we should too...

**Comment by ImALuckyGuy** [comment]
There are a handful of 2-character usernames. I know someone who has the username "i4".

**Comment by JeromSar** [comment]
@ImALuckyGuy That's odd, according to Mojang's account website, you can't have an account with just two characters: account settings

**Comment by OxLemonxO** [comment]
http://www.minecraftforum.net/forums/minecraft-discussion/discussion/182881-2-letter-usernames-no-longer-allowed-to-be-made

**Comment by jjkatz613** [comment]
It might be a cracked account

**Comment by JeromSar** [comment]
@ImALuckyGuy Alright, thanks for contributing. This PR will be merged soon.

**Comment by Wild1145** [comment]
@JeromSar Re-Open?

**Comment by Wild1145** [comment]
@ImALuckyGuy Please check and sign the CLA for this PR to proceed.

**Comment by Wild1145** [comment]
Thank you for your contribution. Sadly we have decided to close this issue as there has been no recent activity to this pull request. If you wish to continue working on this PR please re-open the issue and ensure that all requested changes are made.

**Pull request closed without merging**